scheduler/reconcile.go

```diff
@@ -16,6 +16,11 @@ const (
 	// batchedFailedAllocWindowSize is the window size used
 	// to batch up failed allocations before creating an eval
 	batchedFailedAllocWindowSize = 5 * time.Second
+
+	// rescheduleTimeLapseWindowSize is the window size relative to
+	// current time within which reschedulable allocations are placed.
+	// This helps protect against small clock drifts between servers
+	rescheduleTimeLapseWindowSize = 1 * time.Second
```

Comment:
Not a huge fan of the name/description. Can't think of a better name off the top of my head tho.